docs/section19/19-3.md

```diff
@@ -0,0 +1,26 @@
+# Section 19 - Rule 19-3     
+**Schema Version:** 0.0.23  
+**Mandatory Rule:** False    
+**Rule ID:** 19-2     
+**Rule Description:** Weather conditions used in sizing runs to determine baseline equipment capacities shall be based either on hourly historical weather files containing typical peak conditions or on design days developed using 99.6% heating design temperatures and 1% dry-bulb and 1% wet-bulb cooling design temperatures.    
+**Rule Assertion:** Options are UNDETERMINED     
+**Appendix G Section:** G3.1.2.2.1      
+**90.1 Section Reference:** None  
```

Comment:
Now that committee agreed that some of the rules will be excluded from the scope of 229, I think this rule shuld be one of these because it always applies and cannot be automated.

Reply:
so is the consensus to move this to rules not evaluated?

Reply:
Discussed on call on and decided to make no evaluated.